src/main/java/chess/domain/board/Column.java

```diff
@@ -0,0 +1,49 @@
+package chess.domain.board;
+
+import java.util.Arrays;
+
+public enum Column {
+	A(0, 'a'),
+	B(1, 'b'),
+	C(2, 'c'),
+	D(3, 'd'),
+	E(4, 'e'),
+	F(5, 'f'),
+	G(6, 'g'),
+	H(7, 'h');
+
+	private final int index;
+	private final char column;
+
+	Column(int index, char column) {
+		this.index = index;
+		this.column = column;
+	}
+
+	public static Column findColumn(char input) {
+		return Arrays.stream(values())
+				.filter(value -> value.column == input)
+				.findFirst()
+				.orElseThrow(IllegalArgumentException::new);
+	}
+
+	public static Column findColumnByIndex(int input) {
+		return Arrays.stream(values())
+				.filter(value -> value.index == input)
+				.findFirst()
+				.orElseThrow(IllegalArgumentException::new);
+	}
+
+	public static boolean isInBound(int index) {
+		try {
+			findColumnByIndex(index);
+			return true;
+		} catch (IllegalArgumentException e) {
```

Comment:
보이지 않는 의존성이 존재합니다. 외부에서 사용하는 findColumnByIndex의 예외 스펙이 바뀌게 된다면 엄한 메서드가 오동작할 수 있을거같아요.
```suggestion
		return Arrays.stream(values()).anyMatch(value -> value.index == index);
```
중복을 줄이는것도 좋지만 그것보다는 잠재적 위협을 없애는게 더 우선아닐까요? (따로 예외처리가 없으니 완전 중복은 아닌거같아요)